src/Microsoft.WSMan.Management/WsManHelper.cs

```diff
@@ -1115,7 +1114,6 @@ internal static string GetResourceString(string Key)
         }
 
         /// <summary>
-        ///
```

Comment:
No content in \<summary/\>

Reply:
We have tons empty comments - I can not add useful comments everywhere.

Reply:
We can treat this as a separate issue outside of this PR